PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -501,7 +501,7 @@ void ProjectPanel::buildProjectXml(TiXmlNode *node, HTREEITEM hItem, const TCHAR
 	{
 		tvItem.hItem = hItemNode;
 		SendMessage(_treeView.getHSelf(), TVM_GETITEM, 0, reinterpret_cast<LPARAM>(&tvItem));
-		if (tvItem.lParam != NULL)
+		if (tvItem.lParam != 0)

```

Comment:
Semantic problem: it could be `nullptr` but not `0`.

Reply:
`LPARAM` is not a pointer but an integer. So it's not right to compare it against `NULL`. See my previous [comment](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/10048#discussion_r655494587).